report/report.py

```diff
@@ -126,10 +127,14 @@ def generate_report(args, *images):
     write_report(report)
 
 
-
 def execute_dockerfile(args):
     '''Execution path if given a dockerfile'''
     logger.debug('Setting up...')
+    try:
+        subprocess.check_output('docker ps', shell=True)
```

Comment:
You could use the docker_command wrapper under utils/container.py. It doesn't use shell=True and takes care of things like logging and checking to see if the user doesn't have root privileges.

Reply:
thanks for the suggestion. I will update that..